src/udev/cdrom_id/cdrom_id.c

```diff
@@ -474,8 +471,7 @@ static int cd_profiles_old_mmc(int fd) {
                         cd_media_track_count_data = 1;
                         return 0;
                 } else {
-                        log_debug("no current profile, assuming no media");
-                        return -1;
+                        return log_debug_errno(-1, "no current profile, assuming no media");
```

Comment:
This should be `log_debug_errno(SYNTHETIC_ERRNO(ENOMEDIUM), ...)`. The rules are:
- don't use fake errno values like -1, but the best approximate negative errno code
- if the error value x is received from further up the call stack, use log_debug_errno(x). If it is generated locally for convenience of using log_..._errno(), wrap it in SYNTHETIC_ERRNO().
- there's some code which doesn't follow those rules, it should be made to conform when updating.
- the argument to log_debug_errno() should not have an explicit negative value, i.e. log_debug_errno(r, ...) is OK, but log_debug_errno(-EFOO, ...) should be replaced by log_debug_errno(EFOO, ...).